Library/Formula/libagar.rb

```diff
@@ -0,0 +1,37 @@
+class Libagar < Formula
+  desc "Cross-platform GUI toolkit"
+  homepage "http://libagar.org/"
+  url "http://stable.hypertriton.com/agar/agar-1.4.1.tar.gz"
+  sha256 "b0e62b754f134c3c0dd070a4fa62fa552654356eebab3d55e32d5d9b151a275e"
+  head "http://dev.csoft.net/agar/trunk", :using => :svn
+
+  depends_on "sdl"
+  depends_on "freetype"
+  depends_on "jpeg"
+  depends_on "libpng"
+
+  def install
+    ENV.deparallelize
+    system "./configure", "--prefix=#{prefix}"
+    system "make"
+    system "make", "install", "MANDIR=#{man}" # --mandir for configure didn't work
+  end
+
+  test do
+    (testpath/"test.c").write <<-EOS.undent
+      #include <agar/core.h>
+      #include <agar/gui.h>
+
+      int main() {
+        AG_Window *win;
+        if (AG_InitCore("test", AG_VERBOSE) == -1 || AG_InitGraphics(NULL) == -1) {
+          return 1;
+        } else {
+          return 0;
+        }
+      }
+    EOS
+    system ENV.cc, "test.c", "-o", "test", *`agar-config --cflags --libs`.split
```

Comment:
Well, I didn't want to spell out everything from `agar-config --cflags --libs` gave me:

`-I/usr/local/Cellar/libagar/1.4.1/include/agar -I/usr/local/include/SDL -D_GNU_SOURCE=1 -D_THREAD_SAFE  -I/usr/local/Cellar/freetype/2.6_1/include/freetype2   -I/usr/local/Cellar/libpng/1.6.18/include/libpng16
-L/usr/lib64 -L/usr/local/Cellar/libagar/1.4.1/lib -lag_gui -lag_core -L/usr/local/lib -lSDLmain -lSDL -Wl,-framework,Cocoa -L/usr/local/Cellar/freetype/2.6_1/lib -lfreetype -framework OpenGL -lm -L/usr/local/lib -ljpeg -L/usr/local/Cellar/libpng/1.6.18/lib -lpng16 -lpthread`.

It didn't run without missing dependencies.